executor/slow_query.go

```diff
@@ -62,8 +62,8 @@ type slowQueryRetriever struct {
 	fileLine    int
 	checker     *slowLogChecker
 
-	parsedSlowLogCh chan parsedSlowLog
-	stats           *slowQueryRuntimeStats
+	taskList chan *slowLogTask
```

Comment:
I think rename it to parseTaskList is better. What do you think?

Reply:
I prefer `taskLisk` 😂

Reply:
parse is a verb. `taskList` may be better.